libs/math/fraction.go

```diff
@@ -42,7 +42,8 @@ func ParseFraction(f string) (Fraction, error) {
 		return Fraction{}, errors.New("denominator can't be 0")
 	}
 	if numerator > math.MaxInt64 || denominator > math.MaxInt64 {
-		return Fraction{}, fmt.Errorf("value overflow, numerator and denominator must be less than %d", math.MaxInt64)
+		return Fraction{}, fmt.Errorf("value overflow, numerator and denominator must be less than %s",
+			strconv.FormatInt(math.MaxInt64, 10))
```

Comment:
```suggestion
			math.MaxInt64)
```

does this work? I mean fmt.Errorf(... %s, math.MaxInt64)

Reply:
It overflows as well.

Reply:
The only reason `FormatInt()` works is that the parameter is of type `int64` so it implicitly casts the constant to an int64. It's simpler to just do `fmt.Errorf("... %s", int64(math.MaxInt64))`. 32-bit archs do support 64-bit ints, they just don't default to them.